src/vs/workbench/browser/parts/editor/editorCommands.ts

```diff
@@ -466,6 +474,25 @@ function registerDiffEditorCommands(): void {
 		primary: undefined,
 		handler: accessor => toggleDiffIgnoreTrimWhitespace(accessor)
 	});
+
+	MenuRegistry.appendMenuItem(MenuId.CommandPalette, {
+		command: {
+			id: TOGGLE_BRACKET_PAIR_COLORIZATION_ENABLED,
```

Comment:
@hediet I feel like this command should have a good when clause? What do you think? Like does it make sense to have this command available when you're not looking at an actual editor?